src/ReadTransaction.ts

```diff
@@ -100,7 +100,7 @@ export class ReadTransaction extends Transaction<ReadTransactionRequest, ReadTra
           resolve()
         } else {
           const ret: Instance<T> = {
-            instance: JSON.parse(reply.toObject().instance as string),
+            instance: JSON.parse(fromBase64(reply.toObject().instance as string)),
```

Comment:
We we end up using Buffer here elsewhere anyway, I think we could just stick with Buffer and avoid having those extra b64-lite deps?